.env

```diff
@@ -4,3 +4,5 @@ NEXT_PUBLIC_USE_PACKAGE_VERSION=false
 NEXT_PUBLIC_WALLET_CONNECT_ID=83209157e11b11d87ae68781c8f3762d
 NEXT_PUBLIC_GRAPHQL_SERVER_URL=https://dev-point-api.secured.finance/graphql
 SF_ENV=development
+NEXT_PUBLIC_STABLECOIN_APP_URL=https://dev-stablecoin.on-fleek.app
```

Comment:
@leeshuzheng add these new env variables to the GitHub workflows as well.

Reply:
This is done!